index/scorch/persister.go

```diff
@@ -253,6 +263,7 @@ func (s *Scorch) parsePersisterOptions() (*persisterOptions, error) {
 	po := persisterOptions{
 		PersisterNapTimeMSec:      DefaultPersisterNapTimeMSec,
 		PersisterNapUnderNumFiles: DefaultPersisterNapUnderNumFiles,
+		ActiveMergeMemorySegments: DefaultMergeMemorySegments,
```

Comment:
naming here, if the option is called FooBar, I figured the default should also be named DefaultFooBar?